### ISSUE TITLE
Not compiling on Linux

### ISSUE THREAD

**Issue by skerit** [openc2e/openc2e#194]
I saw in your latest commit c805e437fddd116ef143ee310e04090a18bc053e you added an `<stdint.h>` include to fix ubuntu builds, but I think another one is needed in `fileformats/attFile.h`?

Also: after I add that, I get another compilation error:

```
/home/skerit/projects/openc2e/src/common/SimpleVariant.h:199:30: fout: ‘terminate’ is not a member of ‘std’
  199 |                         std::terminate(); // TODO, throw exception?
      |                              ^~~~~~~~~
/home/skerit/projects/openc2e/src/common/SimpleVariant.h: In memberfunctie ‘const T& SimpleVariant<Ts>::get() const’:
/home/skerit/projects/openc2e/src/common/SimpleVariant.h:207:30: fout: ‘terminate’ is not a member of ‘std’
  207 |                         std::terminate(); // TODO, throw exception?

```

I don't know much about C++, but isn't this part of the standard library? How can it be missing then?
Or is the wrong C++ compiler being configured?

**Comment by ligfx** [comment]
Thanks @skerit for pointing this out. Yep, looks like I need to add that <stdint.h> , and [I'll need to add the <exception> header](https://en.cppreference.com/w/cpp/error/terminate) for std::terminate . I'll add these next time I have some coding time.

**Comment by ligfx** [comment]
Fixed in c805e43 and f2e9d99
